chartfx-chart/src/main/resources/de/gsi/chart/chart-icons.css

```diff
@@ -1,64 +1,61 @@
 @font-face {
-  font-family: 'fair-chart-icons';
-  src: url('fonts/fair-chart-icons.eot?yr8ymj');
-  src: url('fonts/fair-chart-icons.eot?yr8ymj#iefix') format('embedded-opentype'),
-    url('fonts/fair-chart-icons.ttf?yr8ymj') format('truetype'),
-    url('fonts/fair-chart-icons.woff?yr8ymj') format('woff'),
-    url('fonts/fair-chart-icons.svg?yr8ymj#fair-chart-icons') format('svg');
-  font-weight: normal;
-  font-style: normal;
-  font-display: block;
-}
-
-[class^="icon-"],
-[class*=" icon-"] {
-  /* use !important to prevent issues with browser extensions that change fonts */
-  font-family: 'fair-chart-icons' !important;
-  speak: never;
-  font-style: normal;
-  font-weight: normal;
-  font-variant: normal;
-  text-transform: none;
-  line-height: 1;
-
-  /* Better Font Rendering =========== */
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+    font-family: 'fair-chart-icons';
+    src: url('fonts/fair-chart-icons.eot?yr8ymj');
+    src: url('fonts/fair-chart-icons.eot?yr8ymj#iefix') format('embedded-opentype'),
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5780860)